src/router.js

```diff
@@ -80,15 +80,14 @@ export class Router {
   * @param viewPort The viewPort.
   * @param name The name of the viewPort. 'default' if unspecified.
   */
-  registerViewPort(viewPort:Object, name?:string) {
-    name = name || 'default';
+  registerViewPort(viewPort : Object, name? : string = 'default') : void {
```

Comment:
I don't mind the generated code, I just want to make sure that the code in aurelia/templating-router never passes null or empty string and expects the default to be applied. I am not sure what how behavior will change in the cases of `<router-view></router-view>` and `<router-view name=""></router-view>`.